### PR TITLE
Rework of masterdelete class: Reformatting, PHPDoc for methods, removed dead code

### PR DESCRIPTION
**Pull request by andygrunwald**
- No functionality change in methods
- Removed dead code
- Reformatting for proper readability
- Added proper PHPDocs for methods

### CODE REVIEW

File: inc/classes/class_masterdelete.php [hunk 1]
```diff
@@ -125,28 +158,40 @@ public function Delete($table, $idname, $id)
...
+
             unset($_SESSION['md_referrer']);
+
             return $res;
         }
     }
 
+    /**
+     *
+     * TODO Question for ASK_DELETE AND ASK_SET0
```
Comment (M4LuZ) [inline inc/classes/class_masterdelete.php]: there is AFAIK also `@ToDo`f for that

Comment (andygrunwald) [reply inc/classes/class_masterdelete.php]: Yes, many IDEs support both versions: `@todo` or `TODO`.
Afaik there is no strict convention over it.
I mostly search for "todo" which catches both.
I don't have a preference. For me it is just important, to mark this as a task.

Comment (M4LuZ) [reply inc/classes/class_masterdelete.php]: I could only find `@todo` the [PHPdoc documentation](http://docs.phpdoc.org/references/phpdoc/tags/todo.html), although `TODO` is mentioned for inline comments 🤷‍

Comment (andygrunwald) [reply inc/classes/class_masterdelete.php]: Should i change it or do you say "This is fine for now, next time use `@todo` in docblocks and `// TODO` in inline comments?

Comment (M4LuZ) [reply inc/classes/class_masterdelete.php]: I would suggest the second, given that this is a really minor point ;)

**Pull request merged**